server.js

```diff
@@ -98,7 +124,7 @@ try {
 
 (async () => {
   // open location database once at server start. Service includes 24hr check to reload fresh database.
-  await IpLocationService.openLocationDb().catch(e => console.warn(e));
+  await IpLocationService.openLocationDb().catch((e) => console.warn(e));
```

Comment:
Do you like this formatting rule? 😬 Much cleaner to me without the paren for a single arg.  Then again, I prefer single quotes and no semicolons, so this whole repo already fights with my syntax style 😅

Reply:
hah - yeah, I still haven't gotten used to the double quotes vs single - end up doing half and half I think when I write code now...

I actually prefer the most verbose possible (annoyingly) - I like it when everything's written the same (vs some shorthand and some not, so what I would actually write would be:

```
catch((e) => {
  console.warn(e)
});
```

but I'm happy to use whatever you prefer - but we'd just need to change it in the linter / autoformatter level otherwise it'll keep getting changed.

Reply:
The most verbose! 😄 Yes, we are on the opposite sides of syntax spectrum.  I'm fine with your more verbose arrow function syntax.  But maybe we don't need to auto-correct all of mine - we can both live in harmony 😉